manila/share/drivers/vastdata/__init__.py

```diff
@@ -78,6 +89,10 @@ def do_setup(self, context):
         self._host = self.configuration.vast_mgmt_host
         self._vippool_name = self.configuration.vast_vippool_name
         self._root_export = self.configuration.vast_root_export
+        try:
+            self._load_balancing = VipPoolManagementStrategy[str(self.configuration.vast_load_balancing).upper()]
+        except KeyError:
```

Comment:
we should log a warning when the user input in the configuration is invalid (`"%s is invalid - falling back on 'RANDOM' VIP-Pool load-balancing strategy"`)

Reply:
Done